### DISCUSSION TITLE
Kill switch/fail safe trust fund

### DISCUSSION THREAD

**Discussion by cryptod1** [polymorpher/one-wallet#203] [Ideas]
Kill switch/fail safe
If a wallet holder dies suddenly a timer (6 mths+) will start counting down within your wallet.
Log in to reset timer or your wallet or funds will transfer to a nominated:-
* Trustfund charity or single/group beneficiaries.
* Fiat bank account.

This would create a fail safe for investors who wish to pass on assets to non-crypto beneficiaries inc charities.

**Comment by polymorpher** [comment]
Recovery address can serve the first purpose. I don't think expiry time based transfer-to-fiat-bank-account mechanism should not be a functionality of the wallet, especially for the use case described (probate and estate). The person who set up the will can setup a crypto wallet which its key / access should be passed to estate executor / probate attorneys, who can do the crypto-fiat conversion. The said crypto wallet can be the recovery address of the wallet that needs "kill switch / fail safe".

**Comment by polymorpher** [comment]
I will close this for now and move it to discussions
